Packs/CortexXDR/ReleaseNotes/4_2_20.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Palo Alto Networks Cortex XDR - Investigation and Response
+- Improved implementation of *!endpoint* command to use OR filtering instead of AND.
```

Comment:
```suggestion
- **Bracking Changes:** Improved implementation of *!endpoint* command to use OR filtering instead of AND.
```

Reply:
**Breaking Changes:** Improved implementation of *!endpoint* command to use OR filtering instead of AND.